packages/rocketchat-lib/server/methods/sendInvitationSMS.js

```diff
@@ -0,0 +1,64 @@
+import { Meteor } from 'meteor/meteor';
+import { TAPi18n } from 'meteor/tap:i18n';
+import { check } from 'meteor/check';
+import _ from 'underscore';
+
+Meteor.methods({
+	sendInvitationSMS(phones) {
+		const twilioService = RocketChat.SMS.getService('twilio');
```

Comment:
This hard codes twilio as the only sms service?  Are there other services that admins might want to use?  Is it possible to generalize this and pass the desired service into the method somehow?

Reply:
Depending on what I saw, right now in RC there is only option for Twilio as the supported SMS service. They have made it partially modular, where anyone if wants to implement another service can implement it, but currently only Twilio is implemented. If we want to support multiple services, I can add a setting via which you can select which provider to use for inviting purpose, but that is only usable when other SMS services are implemented.

Reply:
So upstream has the ability to configure another SMS service, but no way to use it?  Is that what you mean by "partially modular"?

Is it a lot of work to "add a setting via which you can select which provider to use for inviting purpose"?

Reply:
If it's not a lot of work, I think it's best.  Do you agree?  Just create the option, and use Twilio as the default.  That way it is at least capable of using another service if an admin wants to make another service and register it. Thanks!

Reply:
I'm thinking that we might want to merge this PR and make notes on work like this that we'd like to do as enhancements.  What do you think?

Reply:
Sure, let's do it in another PR.